hermes/probe/delete/delete_file.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Delete_file implements the probe operation for deleting a file in a
+// storage system.
+
+// Package delete implements the file deletion operation with a storage system.
+package delete
```

Comment:
Make the filename match the package name.

Reply:
I've made this change.

Also, go-lint has mentioned that `delete.DeleteFile()` stutters.
I feel that `delete.File()` is strange, but that might just be the go convention.

Do you have any suggestions about how I can alter this?